core/src/main/java/apoc/RegisterComponentFactory.java

```diff
@@ -5,6 +5,7 @@
 import org.neo4j.kernel.extension.ExtensionFactory;
 import org.neo4j.kernel.extension.ExtensionType;
 import org.neo4j.kernel.extension.context.ExtensionContext;
+import org.neo4j.kernel.internal.Version;
```

Comment:
this is internal class for a reason and should not be used here and there for any convience

Reply:
Oh I see. Therefore, please could you tell me what to use as an alternative to `Version.getKernelVersion()`?

Reply:
so there is a dbms.components procedure that will return neo4j version; also here you are expecting that version will also have dot separator and that will be major.minor - there is no guarantee that its the case

Reply:
so there is a dbms.components procedure that will return neo4j version; also here you are expecting that version will also have dot separator and that will be major.minor - there is no guarantee that its the case